### PR TITLE
feat comment

### PR DESCRIPTION
**Pull request by deepak29-git**
1. Add comments
2. Edit comments
3. Delete comments
4. Upvote on comments
5. Downvote on comment

https://user-images.githubusercontent.com/88363801/168468266-fca81857-8ce4-4690-be95-c49deb1e6ac4.mp4

### CODE REVIEW

**Review by arpit01923 — APPROVED** [review]
- clean UI
- good folder structure and naming convention
- impressive code
- one suggestion try absolute paths to avoid ../../

File: src/backend/controllers/CommentsController.js [hunk 1]
```diff
@@ -32,6 +32,50 @@ export const getPostCommentsHandler = function (schema, request) {
...
+//     return new Response(
+//       500,
+//       {},
+//       {
+//         error,
+//       }
+//     );
+//   }
+// };
+
```
Comment (prathmeshjagtap) [inline src/backend/controllers/CommentsController.js:78]: you can remove commented code if not required

**Review by prathmeshjagtap — APPROVED** [review]
The code is Clean and Well Structured.  You can remove commented code if not required

File: src/backend/controllers/CommentsController.js (hunk 1 above)
Comment (deepak29-git) [reply src/backend/controllers/CommentsController.js:78]: ok

**Review by sharathnair9999 — COMMENTED** [review]
looks very clean. good work

**Pull request merged**